common/shared.c

```diff
@@ -992,6 +1025,82 @@ ssize_t ft_inject(struct fid_ep *ep, size_t size)
 	return ret;
 }
 
+ssize_t ft_post_rma(enum ft_rma_opcodes op, struct fid_ep *ep, size_t size,
+		struct fi_rma_iov *remote, void *context)
+{
+	uint64_t cq_data = 1;
```

Comment:
> > +	uint64_t cq_data = 1;
> 
> Why is it odd? Btw, we do have a cq_data initialization function
> ft_init_cq_data which I can reuse here.

This is declaring a variable that's hard set to 1.  Why not just use 1 wherever it's used?  Or why define this at all?